hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -768,6 +778,7 @@ private synchronized CompletableFuture<Void> handleFlushInternalSynchronized(boo
     if (putBlockResultFuture != null) {
       recordWatchForCommitAsync(putBlockResultFuture);
     }
+    clientMetrics.getHsyncSendWriteChunkNs().add(Time.monotonicNowNanos() - start);
```

Comment:
this approach can miss the invocations that throws exception.

Reply:
I think it's ok, we want to see normal metrics. This method is long, I didn't wanna add indentation to it.